src/Features/App/App.tsx

```diff
@@ -110,17 +135,35 @@ export default function App() {
   }
 
   if (hasData) {
+    const feature = featureQuery.data?.features;
+
     return (
-      <FlagsProvider features={{ standaloneModeEnabled: PRODUCT_STANDALONE }}>
+      <FlagsProvider
+        features={{
```

Comment:
Small thing here but potentially could pass the `feature` object as is to the `FlagsProvider` and then in the `FeatureFlag` enum you can have the enum be the following
```
TeamManagement: "team.management"
WorkflowQuotas: "workflow.quotas"
```

Not sure if that would be best but the string literal property access e.g. `feature["team.management"]` and the repeated `Enabled` suffix are a bit of an eyesore to me.